plugins/catalog-graph/src/components/CatalogGraphPage/useCatalogGraphPage.ts

```diff
@@ -56,19 +56,13 @@ export type CatalogGraphPageValue = {
 
 export function useCatalogGraphPage({
   initialState = {},
+  ...props
 }: {
   initialState?: {
     selectedRelations?: string[] | undefined;
     selectedKinds?: string[] | undefined;
-    rootEntityRefs?: string[];
```

Comment:
I may have been a bit overzealous in my original comment. I think this use of `initialState` makes sense for `useCatalogGraphPage` and _probably_ `CatalogGraphPage` as well. I didn't realize we were supporting dynamically setting all of these options.

@benjdlambert is this generally how `initialState` is used across the project?

Reply:
@benjdlambert bump.